### PR TITLE
chore(subgraph,networks): migrate all subgraphs to studio

### LINKED ISSUE #12867: Move all subgraphs to Studio, under the multisig account
We have subgraphs on the hosted version. Let's redeploy them all on the studios infra, thru the multisig account.

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This PR includes several things 

- remove support for the graph hosted version from our scripts
- replace old endpoints by new studio one (all owned by ARB wallets) in networks package
- remove `endpointV2` param from networks package (it is confusing and unconsistent atm)

List of networks migrated 

- [x] arbitrum 
- [x] avalanche
- [x] base
- [x] bsc
- [x] celo
- [x] gnosis
- [x] goerli
- [x] linea ~(not supported by stuudio)~
- [x] mainnet
- [x] mumbai
- [x] optimism
- [x] polygon
- [x] scroll
- [x] sepolia
- [x] zkevm
- [x] zksync (in another PR) 

# Issues

Fixes #12867
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/networks/src/networks/avalanche.ts [hunk 1]
```diff
@@ -62,9 +62,8 @@ export const avalanche: NetworkConfig = {
   startBlock: 17188332,
   subgraph: {
     endpoint:
-      'https://api.thegraph.com/subgraphs/name/unlock-protocol/avalanche',
-    endpointV2:
```
Comment (julien51) [inline packages/networks/src/networks/avalanche.ts:66]: I think the apps still use `endpointV2` though... so we need to change them as well.

File: packages/networks/src/networks/celo.ts [hunk 2]
```diff
@@ -53,11 +53,11 @@ export const celo: NetworkConfig = {
   provider: 'https://rpc.unlock-protocol.com/42220',
   publicLockVersionToDeploy: 13,
   publicProvider: 'https://forno.celo.org',
-  startBlock: 7179039,
+  startBlock: 13994123,
```
Comment (julien51) [inline packages/networks/src/networks/celo.ts:56]: ?? Hum, why this change?

File: packages/unlock-js/src/subgraph/index.ts [hunk 3]
```diff
@@ -33,7 +33,7 @@ export class SubgraphService {
   createSdk(networkId = 1) {
     const network = this.networks[networkId]
 
-    const client = new GraphQLClient(network.subgraph.endpointV2!)
```
Comment (julien51) [inline packages/unlock-js/src/subgraph/index.ts:36]: Oh nice yuou fixed it!

**Review by julien51 — APPROVED** [review]
ok this is very cool! Thank you so much!

**Comment by julien51** [comment]
Actually it looks loke BSC is supported: https://thegraph.com/docs/en/developing/supported-networks/
Screenshot 2024-02-13 at 11 44 09 AM

**Comment by clemsos** [comment]
so far deploying with `bsc` is failing with the following error

```
exec error: Error: Command failed: graph deploy --version-label="v0.0.1" --product=subgraph-studio unlock-protocol-bsc
    Error: The Subgraph Studio only allows subgraphs for these networks: 
    mainnet, rinkeby, goerli, gnosis, chapel, optimism-goerli, clover, fantom,
     matic, fantom-testnet, arbitrum-goerli, fuji, celo-alfajores, mumbai, 
    aurora-testnet, near-testnet, optimism, optimism-goerli, 
    theta-testnet-001, osmo-test-4, base-testnet, base, celo, arbitrum-one, 
    arbitrum-sepolia, avalanche, zksync-era, zksync-era-testnet, sepolia, 
    polygon-zkevm-testnet, polygon-zkevm, scroll-sepolia, scroll
```

**Comment by julien51** [comment]
Are you sure you are using the latest version of the graph cli?

File: packages/networks/src/networks/bsc.ts [hunk 4]
```diff
@@ -86,9 +86,8 @@ export const bsc: NetworkConfig = {
   startBlock: 13079000,
 
   subgraph: {
-    endpoint: 'https://api.thegraph.com/subgraphs/name/unlock-protocol/bsc',
-    endpointV2:
-      'https://api.thegraph.com/subgraphs/name/unlock-protocol/bsc-v2',
+    endpoint: 'https://api.thegraph.com/subgraphs/name/unlock-protocol/bsc-v2',
```
Comment (clemsos) [inline packages/networks/src/networks/bsc.ts:89]:
```suggestion
    endpoint: 'https://api.studio.thegraph.com/query/65299/unlock-protocol-bsc/version/latest',
```
